include/fam/fam.h

```diff
@@ -101,11 +101,25 @@ typedef long long int128_t;
 #endif // __int128t
 #endif // int128_t
 
+typedef enum {
+    INTERLEAVE_DEFAULT,
```

Comment:
we can set the default enum value as zero for all the enum parameters.
INTERLEAVE_DEFAULT=0,